tests/e2e/tests/base/llm-revert-check_jboss-eap-quickstarts.test.ts

```diff
@@ -0,0 +1,119 @@
+import { VSCode } from '../../pages/vscode.page';
+import { SCREENSHOTS_FOLDER, MIN } from '../../utilities/consts';
+import { getRepoName, generateRandomString } from '../../utilities/utils';
+import { providerConfigs } from '../../fixtures/provider-configs.fixture';
+import path from 'path';
+import { KAIViews } from '../../enums/views.enum';
+import { getFileImports } from '../../utilities/file.utils';
+/**
+ * Automates https://github.com/konveyor/kai/issues/798
+ Tests that fixes applied by the LLM do not unintentionally revert .
+ *
+ * - Runs migration analysis and applies two specific fixes.
+ * - Captures and compares import statements before and after fixes.
+ * - Ensures earlier fixes are not reverted by later ones.
+ */
+providerConfigs.forEach((config) => {
+  test.describe(`LLM Revertion tests | ${config.model}`, () => {
+    let vscodeApp: VSCode;
+    const randomString = generateRandomString();
+    let profileName = `llm-reversion-${randomString}`;
+
+    const kitchenRepoPath = 'jboss-eap-quickstarts/kitchensink';
+    const memberFileUri = path.resolve(
+      kitchenRepoPath,
+      'src/main/java/org/jboss/as/quickstarts/kitchensink/model/Member.java'
+    );
+    let beforeTestMemberFileImports: string[];
+    let afterFirstFixMemberFileImports: string[];
+    let afterSecondFixMemberFileImports: string[];
+
+    test.beforeAll(async ({ testRepoData }, testInfo) => {
+      test.setTimeout(15 * MIN);
+      const repoName = getRepoName(testInfo);
+      const repoInfo = testRepoData[repoName];
+
+      vscodeApp = await VSCode.open(repoInfo.repoUrl, repoInfo.repoName, repoInfo.branch);
+      await vscodeApp.closeVSCode();
+      // Only analyzing kitchensink to save time vs. full jboss repo
+      vscodeApp = await VSCode.open(undefined, kitchenRepoPath, undefined);
+      await vscodeApp.createProfile(repoInfo.sources, repoInfo.targets, profileName);
+      await vscodeApp.configureGenerativeAI(config.config);
+      await vscodeApp.startServer();
+    });
+
+    test.beforeEach(async () => {
+      const testName = test.info().title.replace(' ', '-');
+      console.log(`Starting ${testName} at ${new Date()}`);
+      await vscodeApp.getWindow().screenshot({
+        path: `${SCREENSHOTS_FOLDER}/before-${testName}-${config.model.replace(/[.:]/g, '-')}.png`,
+      });
+    });
+
+    test('Analyze jboss-eap-quickstarts', async () => {
+      test.setTimeout(30 * MIN);
+      await vscodeApp.waitDefault();
+      await vscodeApp.runAnalysis();
+
+      console.log(new Date().toLocaleTimeString(), 'Analysis started');
+      await vscodeApp.waitDefault();
+      await vscodeApp.getWindow().screenshot({
+        path: `${SCREENSHOTS_FOLDER}/analysis-running.png`,
+      });
+      await expect(vscodeApp.getWindow().getByText('Analysis completed').first()).toBeVisible({
+        timeout: 15 * MIN,
+      });
+    });
+
+    test('Fix "The package javax has been replaced by jakarta"  with default (Low) effort', async () => {
```

Comment:
This can be removed from all tests ` with default (Low) effort` as these tests are not related to specific efforts